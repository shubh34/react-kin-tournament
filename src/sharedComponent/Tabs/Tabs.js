import './Tabs.scss';
import React from 'react';
import PropTypes from 'prop-types';

const Tabs = ({ children }) => (
	<nav className="c-tab">
		{children}
	</nav>
);


Tabs.propTypes = {
	children: PropTypes.node.isRequired,
};

export default Tabs;
