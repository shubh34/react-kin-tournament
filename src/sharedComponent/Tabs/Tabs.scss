.c-tab {
    border: 2px solid;
    border-color: #ffffff4a;
}