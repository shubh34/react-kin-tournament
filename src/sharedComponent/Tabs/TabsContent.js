import './TabsContent.scss';
import React from 'react';
import PropTypes from 'prop-types';

const TabsContent = ({ children }) => (
	<div className="c-tab-content">
		{children}
	</div>
);


TabsContent.propTypes = {
	children: PropTypes.node.isRequired,
};

export default TabsContent;
