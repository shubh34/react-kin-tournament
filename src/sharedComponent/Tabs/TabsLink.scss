@import '../../colors';

.c-tab-button {
    width: 50%;
    background: $lightGray;
    color: white;
    font-weight: bold;
    &--active {
        background-color: $green;
        color: $darkGray;
    }
}