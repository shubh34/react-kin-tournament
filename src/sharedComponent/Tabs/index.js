import Tabs from './Tabs';
import TabsLink from './TabsLink/TabsLink';
import TabsContent from './TabsContent/TabsContent';

Tabs.TabsLink = TabsLink;
Tabs.TabsContent = TabsContent;
export default Tabs;
