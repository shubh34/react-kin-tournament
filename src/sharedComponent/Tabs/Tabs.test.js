
import React from 'react';
import { shallow } from 'enzyme';
import Tabs from './Tabs';

describe('Tabs', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<Tabs>Tabs Children</Tabs>);
		expect(wrapper).toMatchSnapshot();
	});
});
