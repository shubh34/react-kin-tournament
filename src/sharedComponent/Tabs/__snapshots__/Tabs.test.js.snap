// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders without crashing 1`] = `
<nav
  className="c-tab"
>
  Tabs Children
</nav>
`;
